build.sh

```diff
@@ -50,9 +44,24 @@ if [ "${1}x" == "x" ] || [ "${1}" == "--help" ] || [ "${1}" == "-h" ]; then
   echo "              Default: Dockerfile"
   echo "  DOCKER_FROM The base image to use."
   echo "              Default: 'debian:11-slim'"
-  echo "  DOCKER_TARGET A specific target to build."
-  echo "              It's currently not possible to pass multiple targets."
-  echo "              Default: main"
+  echo "  BUILDX_PLATFORMS"
+  echo "            Specifies the platform(s) to build the image for."
```

Comment:
The indentions of these explanations seem to be off compared to the other lines.

Reply:
We could change it to `cat <<END_OF_DOCS` like you did in the other PR.
But I would do that separately and fix the indention then.